### PR TITLE
Add wait option in Seth client

### PR DESCRIPTION
**Pull request by dplumb94**
Signed-off-by: Darian Plumb <dplumb@bitwise.io>

### CODE REVIEW

File: families/seth/src/sawtooth_seth/cli/account_create.go [hunk 1]
```diff
@@ -30,6 +30,7 @@ type AccountCreate struct {
 	Moderator   string `short:"m" long:"moderator" description:"Alias of another account to be used to create the account"`
 	Permissions string `short:"p" long:"permissions" description:"Permissions for new account; see 'seth permissions -h' for more info"`
 	Nonce       uint64 `short:"n" long:"nonce" description:"Current nonce of moderator account" default:"0"`
+	Wait   		int    `short:"w" long:"wait" description:"Number of seconds Seth client will wait for transaction to be committed" default:"0"`
```
Comment (jsmitchell) [inline families/seth/src/sawtooth_seth/cli/account_create.go]: Looks like you've got an extra tab in there

File: families/seth/src/sawtooth_seth/cli/permissions_set.go [hunk 2]
```diff
@@ -32,6 +32,7 @@ type PermissionsSet struct {
 	Address     string `short:"a" long:"address" required:"true" description:"Address of account whose permissions are being changed; 'global' may be used to refer to the zero address"`
 	Permissions string `short:"p" long:"permissions" required:"true" description:"New permissions for the account"`
 	Nonce       uint64 `short:"n" long:"nonce" description:"Current nonce of the moderator account" default:"1"`
+	Wait   		int    `short:"w" long:"wait" description:"Number of seconds Seth client will wait for transaction to be committed" default:"0"`
```
Comment (jsmitchell) [inline families/seth/src/sawtooth_seth/cli/permissions_set.go]: Looks like an extra tab here

Comment (peterschwarz) [reply families/seth/src/sawtooth_seth/cli/permissions_set.go]: Was `gofmt` run on the code?

Comment (dplumb94) [reply families/seth/src/sawtooth_seth/cli/permissions_set.go]: yes, it seemed to pass despite the extra tabs

Comment (dplumb94) [reply families/seth/src/sawtooth_seth/cli/permissions_set.go]: Misunderstood how gofmt worked. I will run it and integrate the changes in the fix

**Pull request merged**